Ronni-ronsbons/week-4/project express-personal-api/server.js

```diff
@@ -0,0 +1,308 @@
+// require express and other modules
+const express = require('express');
+const app = express();
+
+// parse incoming urlencoded form data
+// and populate the req.body object
+const bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({ extended: true }));
+
+// allow cross origin requests (optional)
+// https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  next();
+});
+
+/************
+ * DATABASE *
+ ************/
+const db = require('./models');
+
+// Data
+let profile = {
+  name: 'Ronni',
+  githubUsername: 'ronsbons',
+  githubLink: 'https://github.com/ronsbons',
+  currentCity: 'San Mateo',
+  pets: [
+      {
+          name: 'Fluffy',
+          type: 'rabbit',
+          isAlive: false,
+      },
+      {
+          name: 'Tweety',
+          type: 'bird',
+          isAlive: false,
+      },
+      {
+          name: 'Tiger',
+          type: 'cat',
+          isAlive: false,
+      },
+      {
+          name: 'Holiday',
```

Comment:
I'm glad to see Holiday is still with us. Good job following the directions and keeping this as hard-coded data.